From e75d814930e4bbdb46c80077a7b7ca6f8ff8e16c Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=E8=B5=96=E6=B4=AA=E5=9C=A3?= <laihongsheng@deepin.com>
Date: Fri, 20 Mar 2020 18:15:15 +0800
Subject: [PATCH 06/29] update series

---
 debian/patches/series | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debian/patches/series b/debian/patches/series
index f3026cb..67dcffa 100644
--- a/debian/patches/series
+++ b/debian/patches/series
@@ -6,5 +6,5 @@
 0007-xkb-use-iso-codes-json-file-to-parse-the-iso-codes-d.patch
 0008-keyboard-iso-code-root-should-be-639-3.patch
 #0009-Fix-input-config.patch
-nodisplay.patch
+#nodisplay.patch
 translate-fcitx-to-chinese.patch
-- 
2.20.1

