From 061ca170e2c23001dca36f4dde28155953e9686f Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=E8=94=A1=E5=8E=9A=E8=BE=89?= <caihouhui@uniontech.com>
Date: Sat, 21 Mar 2020 22:24:38 +0800
Subject: [PATCH 07/29] set IMSwitchKey to True

---
 data/config.desc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/data/config.desc b/data/config.desc
index 8bccacb..03b04e9 100644
--- a/data/config.desc
+++ b/data/config.desc
@@ -59,7 +59,7 @@ Advance=True
 
 [Hotkey/IMSwitchKey]
 Type=Boolean
-DefaultValue=False
+DefaultValue=True
 Description=Enable Hotkey to scroll Between Input Method
 
 [Hotkey/IMSwitchIncludeInactive]
-- 
2.20.1

