Description: 修复鼠标移到，任务栏输入法图标上时，图标样式改变问题
Bug:	     https://pms.uniontech.com/zentao/task-view-22252.html
Author:	     niqiang
Reviewed-By: huxiaodong
Last-Update: 2020-05-20

Index: fcitx-4.2.9.19/src/lib/fcitx/ui.c
===================================================================
--- fcitx-4.2.9.19.orig/src/lib/fcitx/ui.c
+++ fcitx-4.2.9.19/src/lib/fcitx/ui.c
@@ -691,6 +691,7 @@ void FcitxUIOnInputFocus(FcitxInstance*
 FCITX_EXPORT_API
 void FcitxUIOnInputUnFocus(struct _FcitxInstance* instance)
 {
+    return;
     if (UI_FUNC_IS_VALID(OnInputUnFocus))
         instance->ui->ui->OnInputUnFocus(instance->ui->addonInstance);
 
